.github/workflows/ci.yaml

```diff
@@ -0,0 +1,97 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Continuous Integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 2.15
```

Comment:
Let's drop the list of branches - we don't have to exclude any branch.

Reply:
Done.

Reply:
It turns out this was actually important, which I totally forgot about. If a fork has this `ci.yaml` (which it, naturally, does), every pushed branch _there_ will also be built _there_, and _here_ too. In other words, every single PR will be built _twice_. 

In this light, how do you think about getting these lines back?

Reply:
I replaced these with a regex, to minimize long-term maintenance.